interactive/dashboard/api.py

```diff
@@ -0,0 +1,1036 @@
+# from types import NoneType
+import re
+from django.shortcuts import render
+from .forms.optionsforms import *
+from .forms.report_bug_form import BugForm
+from .forms.evolvingforms import *
+from .forms.initial_condforms import *
+from .flow_diagram import generate_flow_diagram
+from .flow_diagram import get_simulation_length, get_species
+from .upload_handler import *
+from .build_unit_converter import *
+from .save import *
+from .models import Document
+from django.http import HttpResponse, HttpRequest, HttpResponsePermanentRedirect
+from django.http import JsonResponse, HttpResponseBadRequest
+import os
+from django.conf import settings
+import mimetypes
+from django.core.files import File
+from interactive.tools import *
+import pandas
+import platform
+import codecs
+import time
+from io import TextIOWrapper
+from rest_framework import generics, status, views, permissions
+from rest_framework.response import Response
+from mechanism.reactions import *
+from mechanism.species import *
+from mechanism.network_plot import *
+from plots.plot_setup import *
+from dashboard.forms.formsetup import *
+from plots.plot_setup import *
+from os.path import exists
+from model_driver.session_model_runner import *
+from dashboard.flow_diagram import *
+import logging
+
+# api.py contains all DJANGO based backend requests made to the server
+# each browser session creates a "session_key" saved to cookie on client
+#       - request.session.session_key is a string representation of this value
+#       - request.session.session_key is used to access documents from DJANGO
+
+
+def beautifyReaction(reaction):
+    if '->' in reaction:
+        reaction = reaction.replace('->', ' → ')
+    if '__' in reaction:
+        reaction = reaction.replace('__', ' (')
+        reaction = reaction+")"
+    if '_' in reaction:
+        reaction = reaction.replace('_', ' + ')
+    return reaction
+
+
+class TestAPIView(views.APIView):
+    def get(self, request):
+        return Response({"message": "This is a test API view"})
+
+def log(string):
+    logging.info(string)
```

Comment:
I'm betting you wanted to have a short hand for `logging.info()`, but I would much prefer that you removed this function and call the logger with the debug level directly. This would make updates to log levels in the future unnecessarily hard.

Reply:
I'm thinking I'll add an optional var for the log function so you can log errors too. I sort of had to do this for pep8 styling, it makes it much easier

Reply:
This is not the way. Surely pep8 can handle `logger.info()` and all its variants.

Reply:
i do that already for plenty of functions, it's just easier to have functions as simple as possible -- I've created debug and error functions to log messages that are more than just 'info'

Reply:
I agree with Kyle on this. You should never have to make custom functions just to be able to follow the pep8 style. If you run into a situation like this and think something like this is necessary, first run in past me or Kyle. We can help you come up with a native-language solution.